app/models/businessmatching/BusinessMatchingMsbServices.scala

```diff
@@ -105,6 +105,14 @@ object BusinessMatchingMsbServices {
       (__ \ "msbServices").read(minLengthR[Set[BusinessMatchingMsbService]](1).withMessage("error.required.msb.services")).flatMap(BusinessMatchingMsbServices.apply)
     }
 
+  def formReadsTP
+  (implicit
+   p: Path => RuleLike[UrlFormEncoded, Set[BusinessMatchingMsbService]]
+  ): Rule[UrlFormEncoded, BusinessMatchingMsbServices] =
+    From[UrlFormEncoded] { __ =>
+      (__ \ "msbServices").read(minLengthR[Set[BusinessMatchingMsbService]](1).withMessage("error.required.msb.services.tp")).flatMap(BusinessMatchingMsbServices.apply)
```

Comment:
What is the '1' doing here?

Reply:
Checking that at least one option is selected using the minLengthR